### DISCUSSION TITLE
Can't 'quit' game - Tribal Rage

### DISCUSSION THREAD

**Discussion by Bapablynde** [otya128/winevdm#1361] [Q&A]
Hi all,

I was able to get the game Tribal Rage running fairly flawlessly with this app, but I can't 'quit' the game with either the menu or the hotkey, it simply doesn't register commands. 

Has anyone else had this experienced this kind of behaivor with other programmes and worked out a solution (other than Windows keys / control and alt etc.)?

**Comment by cracyc** [comment]
I think the game is a win32 program.
